### ISSUE TITLE
Accessibility - "Why Develop a Social Compliance System" Screen - "Home" Link Issue

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Ilab-do-good#29]
The "Why Develop a Social Compliance System" screens "Home" bread crumb link does not announce that it is a link so people will not know that it is there.

Need to fix so they know it's the way to get back to the main application screen.

**Comment by adriennerlong** [comment]
I've flagged this as a high priority issue. We will ask the developers to address before 9/11.

**Comment by jfc3-dol** [comment]
@adriennerlong,

Thanks, for getting the developers to fix this item. It's important for those using screen readers.

**Comment by jfc3-dol** [comment]
Thanks, for making the search icon say it's a search button.
